meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -1704,6 +1702,7 @@ CONFIG_NET_VRF=y
 CONFIG_NET_DSA_MICROCHIP_KSZ_COMMON=y
 # CONFIG_NET_DSA_MICROCHIP_KSZ9477_I2C is not set
 CONFIG_NET_DSA_MICROCHIP_KSZ_SPI=y
+CONFIG_NET_DSA_MICROCHIP_KSZ_PTP=y
```

Comment:
I would not recommend to enable KSZ_PTP. This change will cost about 50Mbits/s
Here are iperf3 results with KSZ_PTP enabled:
[ ID] Interval           Transfer     Bitrate         Retr                                                                                               
[  5]   0.00-100.00 sec  7.09 GBytes   609 Mbits/sec  140             sender   
[  5]   0.00-100.04 sec  7.09 GBytes   609 Mbits/sec                  receiver
Here are iperf3 results with KSZ_PTP disabled:
[ ID] Interval           Transfer     Bitrate         Retr                                                                                               
[  5]   0.00-100.00 sec  7.68 GBytes   660 Mbits/sec  186             sender     
[  5]   0.00-100.04 sec  7.68 GBytes   659 Mbits/sec                  receiver

Reply:
Thanks for the tip and measurements. I've disabled the feature in the newest version of this PR.